database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Build/84_DML_LEASE_PROPERTY.sql

```diff
@@ -0,0 +1,1317 @@
+INSERT INTO [dbo].[PIMS_PROPERTY]([PID], [PIN], [PROPERTY_TYPE_CODE], [PROPERTY_STATUS_TYPE_CODE], [PROPERTY_DATA_SOURCE_TYPE_CODE], [PROPERTY_DATA_SOURCE_EFFECTIVE_DATE], [PROPERTY_CLASSIFICATION_TYPE_CODE], [PROPERTY_TENURE_TYPE_CODE], [NAME], [DESCRIPTION], [ADDRESS_ID], [REGION_CODE], [DISTRICT_CODE], [PROPERTY_AREA_UNIT_TYPE_CODE], [LAND_AREA], [LAND_LEGAL_DESCRIPTION], [ENCUMBRANCE_REASON], [IS_SENSITIVE], [IS_OWNED], [IS_PROPERTY_OF_INTEREST], [IS_VISIBLE_TO_OTHER_AGENCIES], [ZONING], [ZONING_POTENTIAL], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY])
```

Comment:
The address appears to start at 1, so using an address id of 0 here causes all of these inserts to fail.

Reply:
I'll need clarification on this.

Reply:
So, when an address is inserted, it gets an id value provided by the sequence. The current address sequence seems to start at 1, not 0. In this script it seems you are assuming that an address id of 0 exists. However, because the sequence starts at 1 no value of 0 will ever exist, and so the insert fails with foreign key errors.

Reply:
![image](https://user-images.githubusercontent.com/41091511/132886878-8df784ab-70b4-4af3-86cd-7c2e246a0f19.png)

Reply:
@dfilteau is the above issue another delta between your version of the database and the one created by EF? if so I should log another issue to investigate.